src/frontend/README.md

```diff
@@ -1,7 +1,121 @@
-# frontend
+# frontend service
 
-Run the following command to restore dependencies to `vendor/` directory:
+The **frontend** service is responsible for rendering the UI for the store's website.
+It serves as the main entry point for the application routing requests to their 
+appropriate backend service.
+The application uses Server Side Rendering (SSR) to generate HTML consumed by 
+the browser.
```

Comment:
Non blocking for this pr but this is where I envision some sort of standardized feature table to give an at a glance overview

Reply:
I re-organized the readme, and took an initial attempt at a feature table

Reply:
My initial thought is something more like: 

|Signal | Instrumentation library | Custom spans | Custom span attributes |
|-------|---------------------------|--------------|------------------------|
|Traces| X                         |              | X                      |         X                               |
|Metrics| O                         |    na          | na                 |               na                         |
|Logs| O                         |         na     | na                      |              na                         |

Or we invert the matrix & have signals up top then a list of features in row. We can move this to a separate PR to unblock all the other great updates

Reply:
Updated the matrix per the recommendation above